static/js/main.js

```diff
@@ -42,13 +42,47 @@ $('#CollapseToggleHide').click(function(){
 });
 
 
-//Implementation of Light Mode. Could be handled in a more modern way but it does it's job.
+//Themes. Could be handled in a more modern way but it does it's job.
+$oldbody = null;
+$oldtext = null

```

Comment:
this prevents the script from not working because of a reference error in `.removeClass`